BUILD.tools

```diff
@@ -48,3 +49,8 @@ jar_library(
   name='kythe-indexer',
   jars = [jar(org='kythe', name='indexers/java_indexer', rev='v0.0.26-snowchain3')]
 )
+
+jar_library(
+  name = 'scalac-plugin-dep',
+  jars = [jar(org='org.scalameta', name='scalahost_{}'.format(SCALA_REV), rev='1.8.0')],
+)
```

Comment:
Would it be possible to experiment with pre-release versions of scalameta without forking pants?

Reply:
After our recent release, this should now be `jars = [jar(org='org.scalameta', name='scalac-semanticdb_{}'.format(SCALA_REV), rev='2.0.0')],`.